app/javascript/components/ui/icon/icon-component.jsx

```diff
@@ -3,15 +3,20 @@ import PropTypes from 'prop-types';
 
 import './icon-styles.scss';
 
-const Icon = ({ icon, className }) => (
-  <svg className={`c-icon ${className}`} viewBox={icon.viewBox || '0 0 32 32'}>
+const Icon = ({ icon, className, onClick }) => (
+  <svg
+    className={`c-icon ${className}`}
+    viewBox={icon.viewBox || '0 0 32 32'}
+    onClick={onClick}
```

Comment:
![but why](https://media.giphy.com/media/1M9fmo1WAFVK0/giphy.gif)

Reply:
So I can click the Icon.

Reply:
I'd wrap the icon in a `<button>` and add the listener there. Adding it to the svg is weird, I'm sure it not super accesible